PocketKit/Sources/Analytics/AppEvents/Home.swift

```diff
@@ -0,0 +1,229 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+public extension Events {
+    struct Home {}
+}
+
+public extension Events.Home {
```

Comment:
@JohnOberhauser Check out this file, added a ton of events I saw we were not tracking at all.